Wordle.UI/IWordleUI.cs

```diff
@@ -0,0 +1,12 @@
+using Wordle.Domain;
+namespace Wordle.UI;
+
+public interface IWordleUI
```

Comment:
does the interface name need the `I`? the ide already exposes the type.  Unless this is DotNet best practice?

Reply:
I did doublecheck this, but it's a [.net naming convention](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces) to distinguish interfaces from classes with the `I`!

Reply:
I figured, I don't love it, but I understand. 😀